packages/mint-components/src/components/sqm-referral-table/sqm-referral-table-view.tsx

```diff
@@ -1,7 +1,6 @@
 import { h, VNode } from "@stencil/core";
 import { TextSpanView } from "../sqm-text-span/sqm-text-span-view";
```

Comment:
Is this view actually still used? Or has it been replaced with the generic table view?

Reply:
Not used, removed!